reana/reana_dev/cluster.py

```diff
@@ -319,15 +317,21 @@ def job_mounts_to_config(job_mounts):
     helm_install = "cat <<EOF | helm install reana helm/reana -n {namespace} --create-namespace --wait -f -\n{values}\nEOF".format(
         namespace=namespace, values=values_dict and yaml.dump(values_dict) or "",
     )
-    cmds = [
-        "helm dep update helm/reana",
-        helm_install,
-        "kubectl config set-context --current --namespace={}".format(namespace),
-        os.path.join(
-            get_srcdir("reana"),
-            f"scripts/create-admin-user.sh {instance_name} {admin_email} {admin_password}",
-        ),
-    ]
+    cmds = []
+    if mode in ("debug"):
+        cmds.append("reana-dev python-install-eggs")
```

Comment:
Inverting the commands, eggs before submodule update, does the trick! :+1: :partying_face: :tada:

Reply:
> cli: fix cluster-deploy 

You could also call the commit headline "cli: fix cluster-deploy debug mode"